rfcs/0024-add-oneflow-frontend.md

```diff
@@ -0,0 +1,134 @@
+- Feature Name: (`add oneflow frontend`)
+- Start Date: (2021-8-20)
+- RFC PR: [apache/tvm-rfcs#0024](https://github.com/apache/tvm-rfcs/pull/24)
+- GitHub Issue: [apache/tvm#8804](https://github.com/apache/tvm/issues/8804)
+
+# Summary
+[summary]: #summary
+
+To enhance the compatibility of TVM with deep learning frameworks,
+we have created a frontend for TVM that targets [oneflow](https://github.com/Oneflow-Inc/oneflow) 
+
+# Motivation
+[motivation]: #motivation
+
+OneFlow, an open source deep learning framework with whole new frame design and the world's leading technology for distributed system. Here are advantages of OneFlow:
+
+- Perfectly support container platforms(k8s & docker)
+- Handle large models easily
+- Almost zero runtime overhead & linear speedup
+- Support automatic mixed precision
+- ...
```

Comment:
Is this just a placeholder? We should fill out the rest of the motivations or remove the ellipses.

Reply:
Yes，I will delete it。

Reply:
> 
> 
> Perhaps this isn't an issue that is covered in the scope of this RFC, but I'm wondering how independent the importers are from one another, and if it's worthwhile to look at a more general framework for third-party importers.
> 
> Are we aiming to support all operators? What does the mapping from OneFlow operators to TVM look like? Will there be gaps that a user will have to be concerned with? What is the plain for maintenance of the new format to import?

Yes, we will support all commonly used operators. OneFlow will release version 1.0 soon, and our operator interface is aligned with the Pytorch API.